### PR TITLE
Specify dates of Thursday-Monday for show

### PR DESCRIPTION
**Pull request by jakevdp**
Hey – looks like a good show. I added the dates so people would know when it's happening.

### CODE REVIEW

**Comment by rdottle** [comment]
:+1:

**Comment by mdrouhard** [comment]
👍

**Comment by myeong** [comment]
no way

File: upcomingshow.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 Sundown at the Devil's House
 Cafe Nordo
-Next Thursday through Monday!
\ No newline at end of file
+Next Thursday June 23rd through Monday June 27th, 2016!
```
Comment (vsivsi) [inline upcomingshow.txt:3]: Wait, isn't it until the 28th?

Comment (jakevdp) [inline upcomingshow.txt:3]: Um, let me check

Comment (jakevdp) [inline upcomingshow.txt:3]: no.

**Comment by csjohns** [comment]
Thanks for the changes!

**Pull request merged**